Firestore/Protos/build-protos.sh

```diff
@@ -52,45 +52,36 @@
 #   OUT_DIR - The output directory for the generated protos. A subdirectory will
 #     be created for each variant (eg objc, cpp, nanopb)
 
-if [ $# -ne 5 ] ; then
+if [ $# -ne 4 ] ; then
```

Comment:
You also need to not pass in the removed parameter.

i.e. remove Firestore/Protos/CMakeLists.txt:178 (which, again, github won't let me comment on.)

Reply:
Done, thanks. Note that #1780 deletes this script altogether.